modules/appeals_api/app/controllers/appeals_api/v0/appeals_controller.rb

```diff
@@ -11,13 +11,20 @@ class AppealsController < ApplicationController
 
       def index
         appeals_response = Appeals::Service.new.get_appeals(user)
+        log_response(appeals_response)
         render(
           json: appeals_response.body
         )
       end
 
       private
 
+      def log_response(appeals_response)
```

Comment:
Wondering if we want to take better advantage of structured logger, like used in vba_documents/app/workers/upload_processor:
```
      Rails.logger.info('VBADocuments: Submission success',
                        'uuid' => metadata['uuid'],
                        'source' => metadata['source'],
                        'docType' => metadata['docType'],
                        'pageCount' => page_total,
                        'pdfCount' => pdf_total)
```
Makes those fields potentially searchable in a structured way.